packages/ag-charts-enterprise/src/features/annotations/annotationUtils.ts

```diff
@@ -100,7 +100,7 @@ export function invertCoords(coords: Coords, context: AnnotationContext) {
     return { x, y };
 }
 
-export function invert(n: Coords['x' | 'y'], context: AnnotationAxisContext) {
+export function invert(n: Coords['x' | 'y'], context: AnnotationAxisContext | _ModuleSupport.AxisContext) {
```

Comment:
And same here, narrowed to pick `'continuous' | 'scaleBandwidth' | 'scaleInvert' | 'scaleInvertNearest'`.